airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py

```diff
@@ -55,7 +55,7 @@ def get_google_ads_client(credentials) -> GoogleAdsClient:
         try:
             return GoogleAdsClient.load_from_dict(credentials, version=API_VERSION)
         except exceptions.RefreshError as e:
-            message = f"Config Error, Please Check permissions: {str(e)}"
+            message = "The authentication to Google Ads has expired. Re-authenticate to restore access to Google Ads."
```

Comment:
Can we additionally return the `e` as that could contain valuable details about the error?

Reply:
we use 
``` python
raise AirbyteTracedException from e
```
 on the next line